src/main/java/org/mskcc/oncokb/curation/web/rest/ApiProxy.java

```diff
@@ -0,0 +1,53 @@
+package org.mskcc.oncokb.curation.web.rest;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import javax.servlet.http.HttpServletRequest;
+import org.mskcc.oncokb.curation.service.ApiProxyService;
+import org.mskcc.oncokb.curation.web.rest.errors.BadRequestAlertException;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.converter.StringHttpMessageConverter;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.server.ResponseStatusException;
+
+@RestController
+@RequestMapping({ "/api", "/legacy-api" })
```

Comment:
proxy all `/api` as well?

Reply:
I thought we wanted all of core's API's. Do we just want the /legacy-api?